pkg/gce-pd-csi-driver/controller.go

```diff
@@ -20,18 +20,18 @@ import (
 	"math/rand"
 	"sort"
 	"strings"
+	"sync"
 	"time"
 
 	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/meta"
+	b "github.com/cenkalti/backoff/v4"
```

Comment:
Can't you use Backoff from k8s apimachinery/.../wait? The ExponentialBackoff bit won't work as it's based on timers, but Backoff.Step() seems about the same as cenkalti.NextBackoff that you use below.

Note this will save us doing an open source review before deploying in GKE.

Reply:
Ack

Reply:
using the client-go flowcontrol lib now